packages/server/src/channels/teams/conduit.ts

```diff
@@ -72,6 +72,8 @@ export class TeamsConduit extends ConduitInstance<TeamsConfig, TeamsContext> {
         await turnContext.sendActivity(message)
       })
     }
+
+    await this.receive(turnContext, true)
```

Comment:
We send the event after sending the proactive message to teams?

Reply:
The proactive message implementation above is a legacy implementation. The new way of doing this is receiving the proactive trigger in botpress and sending a message. We keep the old one as well to avoid breaking people's stuff.